examples/analog_out/cont_gen_voltage_wfm_int_clk_non_regen.py

```diff
@@ -0,0 +1,54 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a continuous periodic
+waveform with new data using an internal sample clock while
+the task is running.
+"""
+
+import time
+
+import numpy as np
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, RegenerationMode
+
+
+def get_sine_wave_data(cycle):
+    """Generate a sine wave data."""
+    frequency = (10, 50)
+    amplitude = (1, 3)
+    sampling_rate = (100, 500)
```

Comment:
you can change sampling_rate in the middle of a task, but you're not doing that

```suggestion
    sampling_rate = (100, 100)
```

Reply:
You don't need to change the signal parameters to demonstrate non-regen. You just need to output a rate that does not evenly divide the sample clock rate.